comp/core/gui/guiimpl/views/private/js/javascript.js

```diff
@@ -19,10 +19,51 @@ function sendMessage(endpoint, data, method, callback, callbackErr){
     type: method,
     data: data,
     success: callback,
-    error: callbackErr
+    error: function(requestObject, error, errorThrown) {
+      try {
+        callbackErr(requestObject, error, errorThrown)
+      } catch (error) {
+        console.error(error)
+      }
+      $("#agent_status").html("Not connected<br> to Agent");
+      $("#agent_status").css({
+        "background": 'linear-gradient(to bottom, #c62d1f 5%, #f24437 100%)',
+        "background-color": '#c62d1f',
+        "border": '1px solid #d02718',
+        "text-shadow": '0px 1px 0px #810e05',
+        'left': '-180px'
+      })
+      // erase right side of the page
+      $(".page").hide();
+      $(".right").html("");
+      setError(requestObject.status, requestObject.responseText)
+    }
   })
 }
 
+// setError print the 
+/**
+ * Sets the error message and displays it on the page.
+ * @param {number} status - The status code of the error.
+ * @param {string} message - The error message.
+ */
+function setError(status, message) {
+  if (status == 0) {
+    message = "Unable to contact the Datadog Agent. Please ensure it is running."
+  }
+  else if (status == 401) {
+    message = `Not logged in. Please log in to access the Datadog Agent Manager. (initial message: ${message.trim()})`
```

Comment:
```suggestion
    message = `Not logged in. Please ensure that your GUI session has not expired. (initial message: ${message.trim()})`
```

Reply:
Made in 6b75a44bd4062c7ca00dbdf98769f381ca52644d